pkg/encoding/v1/encoding.go

```diff
@@ -219,11 +222,81 @@ func (m *Mount) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+type Health struct {
+	ReadinessProbe interface{} `yaml:"readinessProbe,omitempty"`
```

Comment:
Does this have to be an empty interface?
What is the difference between `ReadinessProbe` and `Readiness`?